src/main.rs

```diff
@@ -9,19 +9,23 @@ use ytdlp_gui::{git_hash, logging, Config, YtGUI};
 fn main() -> iced::Result {
     let mut args = std::env::args();
 
+    let mut url: String = "".to_string();
     if let Some(arg) = args.nth(1) {
         if arg == "--help" || arg == "-h" {
             println!("Usage: ytdlp-gui <OPTIONS>\n");
             println!("Options:");
             println!("-h, --help     Print help");
             println!("-V, --version  Print version");
+            println!("-u, --url      Open video url");
```

Comment:
this should be more descriptive, maybe something like "Starts the application with the provided URL as the download URL"